pkg/webhook/controlplane/ensurer_test.go

```diff
@@ -253,13 +224,6 @@ var _ = Describe("Ensurer", func() {
 			}
 		})
 
-		It("should add missing elements to kube-controller-manager deployment (k8s < 1.27)", func() {
-			err := ensurer.EnsureKubeControllerManagerDeployment(ctx, eContextK8s126, dep, nil)
-			Expect(err).To(Not(HaveOccurred()))
-
-			checkKubeControllerManagerDeployment(dep, "1.26.0")
-		})
-
 		It("should add missing elements to kube-controller-manager deployment (k8s >= 1.27)", func() {
```

Comment:
`EnsureKubeControllerManagerDeployment` no longer does something based on the K8s version.

```suggestion
		It("should add missing elements to kube-controller-manager deployment", func() {
```